about.md

```diff
@@ -58,7 +55,12 @@ Though the majority of our maps and games revolve around World War II, we also h
 * Abstract Maps, like things on hex maps
 * Futuristic games, like Star Wars, or Zombieland
 
-### User Created Content:
+## User Created Content:
 * TripleA contains the ability to create and download user made maps.
 * Many of these maps, like "New World Order" have generated quite a following.
 * Rules can vary by map, depending on what the map creator wants.
+
+## Software Licensing
+
+TripleA is released under the GNU Public License (GPL). This license means you are free to distribute this work and any derivatives you make from this work either commercially or non-commercially but only under the terms of the GPL (i.e. must make source code available to your users and must allow your users themselves to freely distribute the work and any further derivatives under the GPL).
+
```

Comment:
Related to #235, it would probably be a good idea to state the GPL version in this paragraph, as well as providing a link to the full license text.  This can all be jammed into the first sentence:

> TripleA is released under the [GNU General Public License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html) (GPL) version 2 or later.